docs: eludris detailed (#2)
eludris/docs: src/detailed/introduction.md

> # Eludris Detailed

Reviewer: add padding space after this line, also call this `README.md`

> Dedicated and Detailed rundown of what we *plan* Eludris to eventually look like.

Reviewer: ```suggestion
Dedicated and detailed rundown of how we *plan* Eludris to eventually function.
```

> This is mostly sketched out from and is and extension of the old `plan.md` in our `eludris/eludris` repository.

Reviewer: yeet this line

## Overview

> The goal is to provide a uniquely fresh but not totally new experience focused around any topic or interest whilst having a policy of being secure, free (as in monetary value), free (as in freedom), privacy respecting, open source, and decentralized without putting users who may not have knowledge or anything to do with the aforementioned topics at an inconvenience.

Reviewer: commas (it's my fault I know)

Reviewer 2: Yeah, this really should be split up into more than one sentence.

> In addition, as with anything Eludris-related, modifying the source code or making your own clients, libraries, and tooling around Eludris is more than welcome.

Reviewer 2: Extremely nitty, but I'd use `--` (or an actual em-dash) for the interjection just to reduce the number of commas in this section. I think that should make it slightly easier to read.

> In addition, as with anything Eludris-related, modifying the source code or making your own clients, libraries, and tooling around Eludris us more than welcome.

Reviewer: ```suggestion
In addition, as with anything Eludris-related, modifying the source code or making your own clients, libraries, and tooling around Eludris is more than welcome.
```

> In addition, as with anything Eludris-related, modifying the source code or making your own clients, libraries, and tooling around Eludris us more than welcome.

Reviewer: opportunity to mention eludris/awesome and eludris-community

## Takeaways

### You can make Communities.

Communities can be either message-based, post-based, or both.

> Message-based communities work like how a Discord server does, having many different channels secluded to their own types and members can -as you can guess- create messages within channels.

Reviewer 2: I would split this into two sentences, too:

Message-based communities work like how a Discord server does, having many different channels secluded to their own types. Members can -as you can guess- create messages within channels.

Post-based communities work like how a Subreddit does with member being able to create different types of posts, vote on them and leave comments.

Both community types have shared features however, like roles, nicknames and so on.

Communities can be either public or private.

> Communities can get manually reviewed by a staff member to get verified if requested by the community's owner.

Reviewer: ```suggestion
Communities can *usually* (depends on the instance) get manually reviewed by an instance admin to get verified if requested by the community's owner.
```

> Verified communities can claim a namespace getting their own URL invite and are indexed in a list to be easily discover-able, doing that however adds more restrictions upon them however (like no End-To-End-Encryption, stricter moderation and so on).

Reviewer 2: Maybe something like:

Verified communities can claim a namespace getting their own URL invite and are indexed in a list to be easily discoverable. However, doing so adds more restrictions upon them (such as no End-To-End-Encryption, stricter moderation, and so on).

?
This section doesnt really read well as-is, imo.

> A community can get unverified at any point in time if they break the Eludris EULA, Terms of Service or instance rules, that however can be appealed and, unjustified unverification will (hopefully) not happen.

Reviewer: remove this line, there is no eludris EULA, there is no TOS, breaking instance rules should be obvious but even then we have no hand in moderating that

### Accounts are unique.

Accounts for a **single** instance are unique, but when [federated](./federation#overview), this is broken.

You can follow people or send them friend requests.

You can also gain a Reddit Karma like form of point creatively called Social Credit by the Eludris Team, you gain it by getting more up-votes on your posts, spending time interacting with people, getting rewarded by instance moderators or through events.

### Bots are done right.

> Bots will be user accounts like you'd see on Reddit, or Twitter, which would mean that they wouldn't have any bot-specific API limitations, however you have to set your bot as one for it to get verified.

Reviewer 2: I'd probably remove the comma between reddit and twitter, and put the however section in its own sentence.

> Verification for bots only means that the bot and it's owner will be given a little neat badge of honour.

Reviewer: add a note that said verification is done by the respective instance owner

Reviewer 2: "[...] the bot and **its**\* owner [...]"

> Discord style application commands will be available, however unlike discord they will not be forced upon people and will have more utilities, uses and will be more flexible, additionally buttons and more message components will be available to give bot developers more room and tools to make cool stuff.

Reviewer 3: ```suggestion
Discord-styled application commands will be available, however unlike Discord they will not be forced upon people and will have more utilities, uses and will be more flexible, additionally buttons and more message components will be available to give bot developers more room and tools to make cool stuff.
```

> Discord-styled application commands will be available, however unlike Discord they will not be forced upon people and will have more utilities, uses and will be more flexible, additionally buttons and more message components will be available to give bot developers more room and tools to make cool stuff.

Reviewer 2: Same thing here, this should not be one singular sentence.
